samples/hello_world/README.rst

```diff
@@ -10,6 +10,7 @@ prints 'Hello World' to the console. This application can be built into modes:
 
```

Comment:
I actually dislike using hello world to test this, hello world should be kept at simple as possible, we should put this somewhere else, even in a new dedicated test where we can verify other similar features.